documentation/c01-whatis.sil

```diff
@@ -82,11 +82,105 @@ SILE does not benefit from the large ecosystem and community that has grown up a
+		In other words almost anything can be integrated.}
+	This includes the parts documented to be user-facing API surfaces; but also the core modules and other bits the SILE developers generally consider as private APIs.
+	This is a blessing and a curse: you can do anything you want, but also anything that can be touched can be broken.
+	As such it is no secret that SILE is both more flexible more brittle than Typst.
+}
+
+\item{\strong{Market usage:}
+	The Typst’s developers’ background in academia has heavily influenced how and where it is used.
+	The largest demographic is almost certainly university students writing papers for their math and science curriculum, with some journal publishing thrown in for good measure.
+	SILE’s active developers are book publishers and the majority of the work it has grown up around is literature and languages.
+	Its flexibility also makes it an ideal tinkerer’s tool: used in font proofing and other workflows unrelated to book publishing.
+}
+
+\item{\strong{Speed:}
+	Typst is faster than SILE.
+	Full stop.
+	This may matter a lot when you’re making lots of changes to detailed formulas and need to review previews.
+	This may matter less if you work on a book design first with minimal content, copy-edit the actual content separately, then just combine the two in a final step.}
+
+\item{\strong{Platform support:}
+	Typst’s pure Rust implementation with WASM-only plugins makes it easy to compile and run for many platforms including Linux, macOS, Windows, and even the web.
+	SILE’s approach to programming with system access does require a bit more “wiring” to get working on a platform, which currently limits it to use on Linux and macOS.
+	Windows support is certainly possible, although to date has not been published.}
+
+\end{itemize}
+
+Another alternative worth mentioning is Speedata Publisher.
+Speedata is a bit of a Unicorn: using much of LuaLaTeX under the hood while being neither a LaTeX engine nor a wrapper.
+Its main use case is taking XML data and laying it out programmatically; and the emphasis is on data handling.
+If you are trying to typeset a product catalog from a database or data-rich newsletters using fancy templates it might be worth a look.
+
+Most other alternatives are based in one way or another on browser technology.
+WeasyPrint and some others\footnote{PDFreactor, Prince, Antenna House, Typeset.sh, Vivliostyle, BFO Publisher, etc.} take HTML and CSS input and converts them to printable output using web-technology libraries.
+Paged.js and several others\footnote{pdfChip, quarkdown, etc.} do the same by actually shipping the content off to a full browser and asking for print versions back.
+Like paged.js, Quarkdown offloads the actual typesetting to a browser, but uses a home-grown superset of Markdown as its input format instead of HTML plus CSS directly.
```

Comment:
Lots of solutions mentioned here, where the key point is CSS Paged Media support, and other CSS extensions (= it's no longer a "browser", it's a complete Web stack with appropriate extensions). Are you comparing products, or features and technologies? It's a bit in between at places, and quite confusing.